pkg/controller/datavolume-controller.go

```diff
@@ -2565,3 +2578,81 @@ func GetRequiredSpace(filesystemOverhead float64, requestedSpace int64) int64 {
 func newLongTermCloneTokenGenerator(key *rsa.PrivateKey) token.Generator {
 	return token.NewGenerator(common.ExtendedCloneTokenIssuer, key, 10*365*24*time.Hour)
 }
+
+func (r *DatavolumeReconciler) garbageCollect(dataVolume *cdiv1.DataVolume, pvc *corev1.PersistentVolumeClaim) (res *reconcile.Result, err error) {
+	if dataVolume.Status.Phase != cdiv1.Succeeded {
+		return
+	}
+	cdiConfig := &cdiv1.CDIConfig{}
+	if err = r.client.Get(context.TODO(), types.NamespacedName{Name: common.ConfigName}, cdiConfig); err != nil {
```

Comment:
It looks like an innocent and cheap GET so I am not sure, but, maybe this belongs as an env var on the cdi-deployment?

Reply:
Env var to get rid of the Get? so we need to update it whenever CDIConfig DataVolumeTTLSeconds is updated? not sure it worth it.